### PR TITLE
RevoUninstaller.RevoUninstallerPro version 5.0.1

### PR DESCRIPTION
**Pull request by GuyInCanada**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.1 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.1.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/60136)

### CODE REVIEW

File: manifests/r/RevoUninstaller/RevoUninstallerPro/5.0.1/RevoUninstaller.RevoUninstallerPro.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,13 @@
...
+
+PackageIdentifier: RevoUninstaller.RevoUninstallerPro
+PackageVersion: 5.0.1
+Installers:
+- Architecture: x64
+  InstallerType: inno
+  InstallerUrl: https://1baf5346ec1e2437e714-1959a2065c6dc5b4e5e9f6fa5eeb4840.ssl.cf1.rackcdn.com/RevoUninProSetup.exe
+  InstallerSha256: 9F7AD647D126310E850D64F871776462FE359B530CE12CF9155E4E94DE71133A
+ManifestType: installer
+ManifestVersion: 1.1.0
```
Comment (ItzLevvie) [inline manifests/r/RevoUninstaller/RevoUninstallerPro/5.0.1/RevoUninstaller.RevoUninstallerPro.installer.yaml:12]:
```suggestion
PackageIdentifier: RevoUninstaller.RevoUninstallerPro
PackageVersion: 5.0.1
MinimumOSVersion: 10.0.0.0
InstallModes:
- interactive
- silent
- silentWithProgress
Installers:
- InstallerLocale: en-US
  Architecture: x64
  InstallerType: inno
  Scope: machine
  InstallerUrl: https://1baf5346ec1e2437e714-1959a2065c6dc5b4e5e9f6fa5eeb4840.ssl.cf1.rackcdn.com/RevoUninProSetup.exe
  InstallerSha256: 9F7AD647D126310E850D64F871776462FE359B530CE12CF9155E4E94DE71133A
  UpgradeBehavior: install
ManifestType: installer
ManifestVersion: 1.1.0
```

File: manifests/r/RevoUninstaller/RevoUninstallerPro/5.0.1/RevoUninstaller.RevoUninstallerPro.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+ShortDescription: Revo Uninstaller Pro
+Description: Revo Uninstaller helps you to uninstall software and remove unwanted programs easily.
+Moniker: revo-pro
+Tags:
+- revo
+- revo uninstaller
+- cleaner
+ReleaseNotesUrl: https://www.revouninstaller.com/revo-uninstaller-pro-full-version-history/
+ManifestType: defaultLocale
+ManifestVersion: 1.1.0
```
Comment (ItzLevvie) [inline manifests/r/RevoUninstaller/RevoUninstallerPro/5.0.1/RevoUninstaller.RevoUninstallerPro.locale.en-US.yaml:26]:
```suggestion
PackageIdentifier: RevoUninstaller.RevoUninstallerPro
PackageVersion: 5.0.1
PackageLocale: en-US
Publisher: VS Revo Group, Ltd.
PublisherUrl: https://www.revouninstaller.com
PublisherSupportUrl: https://www.revouninstaller.com/support
PrivacyUrl: https://www.revouninstaller.com/revo-uninstaller-privacy-policy
Author: VS Revo Group, Ltd.
PackageName: Revo Uninstaller Pro
PackageUrl: https://www.revouninstaller.com/products/revo-uninstaller-pro
License: Proprietary
LicenseUrl: https://www.revouninstaller.com/revo-uninstaller-pro-eula/
Copyright: Copyright VS Revo Group
CopyrightUrl: https://www.revouninstaller.com/revo-uninstaller-pro-eula/
ShortDescription: Revo Uninstaller helps you to uninstall software and remove unwanted programs easily.
Description: Revo Uninstaller is a freeware uninstall utility. It has powerful features to uninstall programs scanning for leftover files, folders and registry entries after uninstall. With its unique 'Hunter mode' it offers you some simple, easy to use but effective and powerful approaches to manage (uninstall, stop, delete, disable from auto starting) and to get information about your installed and/or running application. Many cleaning tools included!
Moniker: revo-pro
ReleaseNotesUrl: https://www.revouninstaller.com/revo-uninstaller-pro-full-version-history/
Tags:
- revo
- revo-uninstaller-pro
- cleaner
ManifestType: defaultLocale
ManifestVersion: 1.1.0
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Comment by tim-mann** [comment]
The installer URL is from https://www.revouninstaller.com/revo-uninstaller-pro-full-version-history/

**Review by ItzLevvie — DISMISSED** [review]

**Comment by tim-mann** [comment]
The installer URL is from https://www.revouninstaller.com/revo-uninstaller-pro-full-version-history/

File: manifests/r/RevoUninstaller/RevoUninstallerPro/5.0.1/RevoUninstaller.RevoUninstallerPro.installer.yaml [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+PackageIdentifier: RevoUninstaller.RevoUninstallerPro
+PackageVersion: 5.0.1
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+  InstallerLocale: en-US
```
Comment (ItzLevvie) [inline manifests/r/RevoUninstaller/RevoUninstallerPro/5.0.1/RevoUninstaller.RevoUninstallerPro.installer.yaml:12]:
```suggestion
- InstallerLocale: en-US
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Comment by tim-mann** [comment]
The installer URL is from https://www.revouninstaller.com/revo-uninstaller-pro-full-version-history/

**Comment by ItzLevvie** [comment]
An engineer will take a look at this PR.

**Pull request merged**